runhouse/main.py

```diff
@@ -679,6 +303,150 @@ def cluster_list(
+        sky_live_clusters = get_all_sky_clusters()
+        cluster_name = (
+            cluster_name.split("/")[-1] if "/" in cluster_name else cluster_name
+        )
+        if cluster_name in sky_live_clusters:
+            console.print(
+                f"You can keep warm the cluster by running [italic bold] sky autostop {cluster_name} -i {mins}"
+            )
+    except Exception as e:
+        console.print(f"Failed to keep the cluster warm: {e}")
+
+
+@cluster_app.command("up")
+def cluster_up(
+    cluster_name: str = typer.Argument(
+        None,
+        help="The name of cluster to bring up.",
+    )
+):
+    """Bring up the cluster if it is not up. No-op if cluster is already up.
+    This only applies to on-demand clusters, and has no effect on self-managed clusters.
+
+    Example:
+
+        ``$ runhouse cluster up /sashab/rh-basic-cpu``
+
+    """
+    current_cluster = get_cluster_or_local(cluster_name)
+
+    try:
+        current_cluster.up_if_not()
+    except ValueError:
+        console.print("Cluster is not saved in den.")
+    except Exception as e:
+        console.print(f"Failed to bring up the cluster: {e}")
+
+
+@cluster_app.command("down")
+def cluster_down(
+    cluster_name: str = typer.Argument(
+        None,
+        help="The name of cluster to terminate.",
+    ),
+    remove_configs: bool = typer.Option(
+        False,
+        "-rm",
+        "--remove-configs",
+        help="Whether to delete cluster config from Den (default: ``False``).",
+    ),
+):
+    """Terminate cluster if it is not down. No-op if cluster is already down.
+    This only applies to on-demand clusters, and has no effect on self-managed clusters.
+
+    Example:
+
+        ``$ runhouse cluster down /sashab/rh-basic-cpu``
+    """
+    current_cluster = get_cluster_or_local(cluster_name)
+
+    try:
+        current_cluster.teardown_and_delete() if remove_configs else current_cluster.teardown()
+    except ValueError:
+        console.print("Cluster is not saved in den, could not bring it down.")
+        sky_live_clusters = get_all_sky_clusters()
+        cluster_name = (
+            cluster_name.split("/")[-1] if "/" in cluster_name else cluster_name
+        )
+        if cluster_name in sky_live_clusters:
+            console.print(
+                f"You can bring down the cluster by running [italic bold] sky down {cluster_name} -y"
+            )
+    except Exception as e:
+        console.print(f"Failed to terminate the cluster: {e}")
+
+
+@cluster_app.command("logs")
+def cluster_logs(
+    cluster_name: str = typer.Argument(
+        None,
+        help="Name of cluster to load the logs from. If not specified, loads the logs of the local cluster.",
+    ),
+    since: Optional[LogsSince] = typer.Option(
+        LogsSince.five.value,
+        "--since",
+        help="Get the logs from the provided timeframe (in minutes).",
+    ),
+):
+    """Load the logs of the Runhouse daemon running on a cluster.
```

Comment:
i think we should be consistent in how we refer to the server / daemon to avoid any confusion. For the server commands we are calling it "server", so maybe we do the same thing for the cluster commands. wdyt?

Reply:
yeah good point